<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>电子标签编辑器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="init">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>初始化标签</string>
    </property>
    <widget class="QPushButton" name="btn_init">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>开始初始化</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_init">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>411</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="text_init">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>1001</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>310</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_read">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>430</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查看</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>图书编码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_code">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>299</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>书架：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>书格：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>299</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>图书名称：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>备注信息：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>981</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>编辑标签</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>init</zorder>
   <zorder>line</zorder>
   <zorder>btn_save</zorder>
   <zorder>btn_read</zorder>
   <zorder>label</zorder>
   <zorder>edit_code</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>lineEdit_4</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>textEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_card">
    <property name="title">
     <string>读卡器</string>
    </property>
    <addaction name="action_connect"/>
    <addaction name="action_status"/>
    <addaction name="separator"/>
    <addaction name="action_drive"/>
   </widget>
   <widget class="QMenu" name="menu_seting">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_club"/>
    <addaction name="separator"/>
    <addaction name="action_kill"/>
    <addaction name="action_pass"/>
   </widget>
   <widget class="QMenu" name="menu_about">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="separator"/>
    <addaction name="action_me"/>
   </widget>
   <addaction name="menu_card"/>
   <addaction name="menu_seting"/>
   <addaction name="menu_about"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_connect">
   <property name="text">
    <string>连接</string>
   </property>
  </action>
  <action name="action_status">
   <property name="text">
    <string>状态</string>
   </property>
  </action>
  <action name="action_drive">
   <property name="text">
    <string>下载驱动</string>
   </property>
  </action>
  <action name="action_club">
   <property name="text">
    <string>绘本馆</string>
   </property>
  </action>
  <action name="action_kill">
   <property name="text">
    <string>灭活密码</string>
   </property>
  </action>
  <action name="action_pass">
   <property name="text">
    <string>访问密码</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="action_me">
   <property name="text">
    <string>作者</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
